fhir-persistence-schema/src/main/resources/db2/erase_resource.sql

```diff
@@ -80,13 +80,25 @@ BEGIN
     PREPARE d_stmt FROM 'DELETE FROM {{SCHEMA_NAME}}.' || p_resource_type || '_quantity_values     WHERE logical_resource_id = ?';
     EXECUTE d_stmt USING v_logical_resource_id;
 
-    PREPARE d_stmt FROM 'DELETE FROM {{SCHEMA_NAME}}.' || 'str_values          WHERE logical_resource_id = ?';
+    PREPARE d_stmt FROM 'DELETE FROM {{SCHEMA_NAME}}.' || p_resource_type || '_profiles            WHERE logical_resource_id = ?';
```

Comment:
are there "global" tables for profiles and tags that we need to delete from as well, or just these resource-type-specific ones?

Reply:
ah, now i see them down below.  `logical_resource_profiles` and `logical_resource_tags` respectively.